ParabolInc/parabol
# feat: create azure issue

Fix https://github.com/ParabolInc/parabol/issues/7026

https://www.loom.com/share/f4a4a7a97e3c4397a5373782cfa230f0

### To test

- Integrate with Azure DevOps. You may need to run `yarn postdeploy` or add the feature flag if it's not an option in your integration table.
- Go to the Scope phase of a Sprint Poker meeting
- Create an issue using the fab
- Can successfully change the project that the issue is sent to 
- The issue shows up at the top of the results
- Can click on the URL in the newly issue and get taken to that issue

Note: 

- The link in the Azure item says "Basic:Issue" rather than the project name. I suggest we change this and have created an issue: https://github.com/ParabolInc/parabol/issues/7048
- If you create many issues and refresh the page a few times, the integration can timeout. I created an issue for this: https://github.com/ParabolInc/parabol/issues/7047
- After ~24 hours, I need to manually refresh the Azure token to see any results: https://github.com/ParabolInc/parabol/issues/7006
- After selecting an issue in the Scope phase or creating a new issue, the subtitle in the newly created item is an id instead of the issue number. Issue here: https://github.com/ParabolInc/parabol/issues/7051
